ibis/impala/compiler.py

```diff
@@ -1162,7 +1007,8 @@ def _sign(translator, expr):
     ops.DayOfWeekName: _day_of_week_name,
 }
 
-_operation_registry.update(_binary_infix_ops)
+_operation_registry.update(binary_infix_ops)
+_operation_registry.update(base_sql._operation_registry)
```

Comment:
It'd be better to do it the other way round. Start with `base_sql._operation_registry` (which you can make public removing the first underscore), and then updating with the Impala specific backend.

There is probably no overlap, but if something is defined in both, would be good to overwrite the base operation with the impala operation, not the other way round.